src/forms/FormFieldMapCoordinates/MapCoordinatesTypes.ts

```diff
@@ -38,6 +38,12 @@ export type MapCoordinatesDef = {
 	 * Latitude and longitude object.
 	 */
 	mapPosition?: MapPosition;
+	/**
+	 * The Google Maps API provides map tiles at various zoom levels for map 
+	 * type imagery. Most roadmap imagery is available from zoom levels 0 to 18
+	 * https://developers.google.com/maps/documentation/javascript/maxzoom
+	 */
+	zoom?: number;

```

Comment:
Looking at what's expected in the ticket I think we might be missing the default center or default location. It looks we have the default zoom, and mapPosition I'm assuming it's default center, so we might be missing defaultCenter (marker position).